app/status/views.py

```diff
@@ -1,9 +1,50 @@
-from flask import jsonify
+from flask import jsonify, current_app
+import json
 
 from . import status
 from . import utils
+from .. import models
 
 
 @status.route('/_status')
 def status():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    api_response = utils.return_response_from_api_status_call(
+        models.get_api_status
+    )
+
+    search_api_response = utils.return_response_from_api_status_call(
+        models.get_search_api_status
+    )
+
+    apis_wot_got_errors = []
```

Comment:
Same again on the variable name I'm afraid. I'm not against frivolity, honest...